tests/e2e/utils/flows.js

```diff
@@ -130,6 +131,12 @@ const CustomerFlow = {
 		} );
 	},
 
+	goToPaymentMethods: async () => {
```

Comment:
@luizreis I like those flows related utils, well done! However, IMO we should keep them separately from those flows for now because generic flows will be moved to their own package soon (more details in p7bje6-2gV-p2). I hope there will be a way to extend the flows but it will be easier to do once our customizations are groupped. As a suggestions I'd create `PaymentsFlows` variable or something like that.

Reply:
Thanks for bringing this up. I've separated our custom flow from the generic ones in https://github.com/Automattic/woocommerce-payments/pull/816/commits/7991cce7f3e466ad94eba915cea81b268f33c776.